### DISCUSSION TITLE
Attaching any file to an issue

### DISCUSSION THREAD

**Discussion by vcrobe** [community/community#19768] [Projects and Issues]
I think it should be possible to attach any file (with any extension) to an issue.

Currently I'm trying to attach a saved email _.eml_ file to an issue and it's not allowed. However it's allowed to attach a _.pdf_ or _.zip_ 
file.

I usually compress files using 7zip so the files are way much smaller than a zip file. But in this case I'm forced to attach a zip file to the issue.

**Comment by nradov** [comment]
Yes it should be possible to attach a `.eml` file to an issue. That is the standard export file format used by most email clients such as GMail. When I receive an email relevant to a GitHub issue I'd like to be able to export it and attach it directly to the issue for easy reference. I don't want to have to print the email as a PDF.
![Screenshot 2024-08-30 at 2 12 34 PM](https://github.com/user-attachments/assets/e9733015-325a-4bc2-b8f6-9038dcc2236b)
